src/features/settings/settingsSlice.ts

```diff
@@ -11,7 +11,9 @@ import type { Asset } from '../../domain/asset';
 import type { MarketLabelled } from '../../domain/market';
 import { featuredAssets } from '../../utils';
 
-const USE_PROXY = '__TAURI__' in window || process.env.USE_PROXY !== undefined;
+const USE_PROXY =
+  '__TAURI__' in window || ('USE_PROXY' in window && Boolean((window as any).USE_PROXY) === true);
```

Comment:
why not just checking for 'USE_PROXY in window`?

```suggestion
  '__TAURI__' in window || 'USE_PROXY' in window;
```

it doesn't make sense to build or run the app with USE_PROXY=false.. it can just be omitted. If the var is set, then it appears on window and the proxy will be used regardless the actual value of the var.

Reply:
well no, if not in tauri (ie normal web app)  the definition `window.PROXY_URL = __PROXY_URL__` makes sure is always defined. there is no real ENV var, so this is the only trick available

https://create-react-app.dev/docs/title-and-meta-tags/#injecting-data-from-the-server-into-the-page

Reply:
the USE_PROXY=false makes sense for the github action deloyment for example, where the user will not use any proxy and install the certificate manually

Reply:
indeed I MUST "patch" the GH action tha publish indeed